_posts/2016-11-28-mass-update-with-joins.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Bulk Updates with Join Tables in Postgresql"
+author: "Ming Xiang Chan"
+date: 2016-11-28 10:20:15 +0800
+tags: [ruby, postgresql, rails]
```

Comment:
since you've added new tags, need to add additional tag files. Instructions are below: 

When you add tags, check `_tags` folder to make sure `<your-tag>.md` exists. If not, create `<your-tag>.md` file with:
```
---
slug: elixir
name: Elixir
---
```